frontend/src/scenes/events/eventsTableLogic.ts

```diff
@@ -63,9 +83,12 @@ export const eventsTableLogic = kea({
     reducers: () => ({
         // save the pathname that was used when this logic was mounted
         // we use it to NOT update the filters when the user moves away from this path, yet the scene is still active
-        initialPathname: [(state) => router.selectors.location(state).pathname, { noop: (a) => a }],
+        initialPathname: [
+            (((state: Record<string, any>) => router.selectors.location(state).pathname) as unknown) as string,
```

Comment:
This also doesn't seem great but initialPathname kept trying to be a function instead of a string

Reply:
This (reducer defaults as selectors) looks like something that should work, but is missing from typegen. I made an issue about it here: https://github.com/keajs/kea-typegen/issues/13

One alternative for better types could be to make an empty reducer and use the [`defaults` object](https://kea.js.org/docs/guide/additional#defaults) to initially fill it. That should have better type support if given a function.